src/main/webapp/app/exam/manage/exams/exam-exercise-import/exam-exercise-import.component.html

```diff
@@ -0,0 +1,140 @@
+                                    [(ngModel)]="exerciseGroup.isMandatory"
+                                />
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </td>
+            <td>
+                <!-- Display a List of all Exercises to the right -->
+                <table class="table table-hover table-striped text-center">
+                    <thead>
+                        <tr>
+                            <th class="smallbox" jhiTranslate="artemisApp.examManagement.exerciseGroup.importModal.selected">Selected</th>
+                            <th class="smallbox">#</th>
+                            <th class="smallbox" jhiTranslate="artemisApp.exercise.type">Type</th>
+                            <th [class]="exerciseGroupContainsProgrammingExercises(exerciseGroup) ? 'mediumbox' : 'largebox'">
+                                <label jhiTranslate="artemisApp.exercise.title">Title</label>
+                                <jhi-help-icon placement="top" text="artemisApp.examManagement.exerciseGroup.importModal.exerciseTitleTooltip"></jhi-help-icon>
+                            </th>
+                            <th *ngIf="exerciseGroupContainsProgrammingExercises(exerciseGroup)" class="mediumbox">
+                                <label class="label-narrow" jhiTranslate="artemisApp.exercise.shortName">Short Name</label>
+                                <jhi-help-icon placement="auto" text="artemisApp.examManagement.exerciseGroup.importModal.shortNameTooltip"></jhi-help-icon>
+                            </th>
+                            <th class="smallbox" jhiTranslate="artemisApp.exercise.difficulty">Difficulty Level</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr *ngFor="let exercise of exerciseGroup.exercises" [id]="'exercise-' + exercise.id">
+                            <td>
+                                <h4 class="form-check d-flex justify-content-center">
+                                    <input
+                                        type="checkbox"
+                                        class="form-check-input"
+                                        [checked]="exerciseIsSelected(exercise, exerciseGroup)"
+                                        (click)="onSelectExercise(exercise, exerciseGroup)"
+                                    />
+                                </h4>
+                            </td>
+                            <td>{{ exercise.id }}</td>
+                            <td>
+                                <fa-icon size="2x" [icon]="getExerciseIcon(exercise)"></fa-icon>
+                            </td>
+                            <td>
+                                <input
+                                    *ngIf="exercise.type! !== exerciseType.PROGRAMMING"
```

Comment:
```suggestion
                                    *ngIf="exercise.type !== exerciseType.PROGRAMMING"
```

Also not needed for the other occurrences. It's simply a source for Sentry issues ^^